ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/CycleUpdaterThreadFactory.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.hive.ql.txn.compactor;
+
+/**
+ * Factory for the {@link CycleUpdaterThread} class
+ */
+final class CycleUpdaterThreadFactory {
+
+  private static final long DEFAULT_UPDATE_INTERVAL_IN_MILLISECONDS = 1_000L;
```

Comment:
Do we want to make this config costumisable?

Reply:
I was thinking on this while implementing...  I can do that